package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amo-tools-suite",
-  "version": "0.1.34",
+  "version": "0.1.35",
```

Comment:
@gaccawi This was accidentally left around from the npm publish I did. Should this be here or should I leave this unchanged?